### ISSUE TITLE
Language switcher: Provide information about unavailable translations for both sighted and AT users

### ISSUE THREAD

**Issue by smillart** [openfed/kiso#53]
In the following code snippet, the `aria-hidden="true"` is an accessibility violation. This attribute attempts to hide important information for _Assistive Technology (AT)_ users, while providing it to sighted users.

https://github.com/openfed/kiso/blob/7efb90813eb993f76a61c858ce4e9fcb1a4ec399/templates/navigation/links--language-block.html.twig#L27-L29

Provide information to users about missing translations is useless since those are unavailable! What is relevant, it is to output links to available translations only.

**Comment by laurent-b4y** [comment]
Here's the code I used on ABH to solve this issue

in the links--language-block.html.twig

```
{#
/**
 * @file
 * Default Kiso's theme implementation to display the "Language switcher" set of links.
 *
 * You will find same variables as in the core 'links.html.twig' template.
 *
 * Related WCAG resources:
 * - SC 1.3.1 Info and Relationships (Level A):
 *   - H48: Using ol, ul and dl for lists or groups of links.
 *   - H49: Using semantic markup to mark emphasized or special text.
 * - SC 3.1.4 Abbreviations (Level AAA):
 *   - H28: Providing definitions for abbreviations by using the abbr element.
 *
 * @see template_preprocess_links()
 * @see kiso_preprocess_links__language_block()
 */
#}
{{ attach_library('kiso/links--language-block') }}
{% if links -%}
	<ul{{attributes.addClass('links--language-block')}}>
		{%- for item in links -%}

			{%- if item.link -%}
				<li{{item.attributes.removeAttribute('hreflang')}}>
					{{ item.link }}
				</li>
			{%- elseif item.text_attributes -%}
				{# do nothing if there's no translation in this language #}
			{%- else -%}
				<li{{item.attributes.removeAttribute('hreflang')}}>
					{{ item.text }}
				</li>
			{%- endif -%}

		{%- endfor -%}
	</ul>
{%- endif %}

```

**Comment by smillart** [comment]
**Last update:** We have to align the _"Language switcher" (Block)_ behavior for all BOSA websites, meaning that information about missing translations should be provided to both sighted and _Assistive Technology (AT)_ users.
